packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -434,7 +433,7 @@ export function getProveDisplayOptions(): ProveDisplayOptions<ZKEdDSAEventTicket
         argumentType: ArgumentTypeName.PCD,
         description: "Generate a proof for the selected ticket",
         validate(value, params) {
-          if (value.type !== EdDSAPCDTypeName || !value.claim) {
+          if (value.type !== EdDSATicketPCDTypeName || !value.claim) {
```

Comment:
be careful with changes like this - i suggest going through the zk ticket example in `consumer-client` even if you're sure this is the right thing to do

Reply:
It does still work!